akka-sample-cqrs-scala/src/main/scala/sample/cqrs/ShoppingCartRoutes.scala

```diff
@@ -0,0 +1,100 @@
+package sample.cqrs
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.cluster.sharding.typed.scaladsl.ClusterSharding
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import akka.util.Timeout
+
+object ShoppingCartRoutes {
+  final case class AddItem(cartId: String, itemId: String, quantity: Int)
+  final case class UpdateItem(cartId: String, itemId: String, quantity: Int)
+  final case class Checkout(cartId: String)
+}
+
+class ShoppingCartRoutes()(implicit system: ActorSystem[_]) {
+
+  implicit private val timeout: Timeout = 5.seconds
```

Comment:
I'd put this in config as well when you anyway have system in scope

Reply:
done